packages/fxa-settings/src/lib/model-data/data-stores/url-query-data.ts

```diff
@@ -30,7 +30,7 @@ export class UrlQueryData extends UrlData {
    * Sets a new internal state from a set of URL search params
    * @param params
    */
-  protected setParams(params: URLSearchParams) {
+  public setParams(params: URLSearchParams) {
```

Comment:
I wonder if we should leave a comment above this about third party auth being a special case to make this `public`? Just don't want us looking at this method later and using it but 🤷‍♀️